src/components/atoms/calendarIssueHeader/calendarIssueHeader.scss

```
@@ -0,0 +1,21 @@
+.issueHeaderAgenda {
```

```suggestion
.issue-header-agenda {
```
mettre les noms de class sous cette forme